benchmark/benchmark_view_assignment.cpp

```diff
@@ -89,7 +89,9 @@ namespace xt
                 {
                     auto v = xt::view(tens, i, j, all(), all());
                     xt::xtensor<double, 2> vas(std::array<std::size_t, 2>({3, 3}));
-                    std::copy(v.data() + v.data_offset(), v.data() + v.data_offset() + vas.size(), vas.begin());
+                    std::copy(
+                        v.data() + v.data_offset(), v.data() + v.data_offset() + vas.size(), vas.begin()
+                    );
```

Comment:
It does shorten the line length (only a bit here because `std::copy` is already short).
Black will do the first one of the following that match the line length
A -
```cpp
std::super_duper_long_fuction_name(v.data() + v.data_offset(), v.data() + v.data_offset() + vas.size(), vas.begin());
```
B -
```cpp
std::super_duper_long_fuction_name(
    v.data() + v.data_offset(), v.data() + v.data_offset() + vas.size(), vas.begin()
);
```
C -
```
std::super_duper_long_fuction_name(
    v.data() + v.data_offset(),
    v.data() + v.data_offset() + vas.size(),
    vas.begin()
);
```
We can remove option B if you prefer.

Reply:
B is also useful with assignments 
```cpp
long_type_name long_var_name = std::super_duper_long_fuction_name(
    v.data() + v.data_offset(), v.data() + v.data_offset() + vas.size(), vas.begin()
);
```

Reply:
I think the formatting of fuctions should always be the same, whether they are used for assignment or not. I would definitely remove option B.

Reply:
Ok, done.